sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/CheckRestrictedAreas.java

```diff
@@ -0,0 +1,177 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.extension.siddhi.execution.geovelocity;
+
+import org.wso2.extension.siddhi.execution.geovelocity.api.GeoVelocityData;
+import org.wso2.extension.siddhi.execution.geovelocity.api.GeoVelocityDataResolver;
+import org.wso2.extension.siddhi.execution.geovelocity.internal.exception.GeoVelocityException;
+import org.wso2.extension.siddhi.execution.geovelocity.internal.impl.GeoVelocityDataResolverHolder;
+import org.wso2.siddhi.annotation.Example;
+import org.wso2.siddhi.annotation.Extension;
+import org.wso2.siddhi.annotation.Parameter;
+import org.wso2.siddhi.annotation.ReturnAttribute;
+import org.wso2.siddhi.annotation.util.DataType;
+import org.wso2.siddhi.core.config.SiddhiAppContext;
+import org.wso2.siddhi.core.executor.ExpressionExecutor;
+import org.wso2.siddhi.core.executor.function.FunctionExecutor;
+import org.wso2.siddhi.core.util.config.ConfigReader;
+import org.wso2.siddhi.query.api.definition.Attribute;
+import org.wso2.siddhi.query.api.exception.SiddhiAppValidationException;
+import java.util.Map;
+
+/**
+ * This is a class to get the risk based on restricted area combinations.
+ */
+@Extension(
+        name = "restrictedAreaBasedRisk",
+        namespace = "geovelocity",
+        // add more to the description
```

Comment:
Need to fix this

Reply:
Fixed with commit ff598ff6b37e6558eda885f660845c33173135fc